homework/react_2025/react-1-hw-1/app/about_us/components/ourValues/OurValues.jsx

```diff
@@ -0,0 +1,15 @@
+import styles from "./ourValues.module.css";
+import { our_Values } from "@/data/consts/ourValues";
+
+export const OurValues = () => {
+  return (
+    <div className={styles.ourValuesContainer}>
+      {our_Values.map((value) => (
+        <div key={value.id} className={styles.valueCard}>
+          <h3 className={styles.valueTitle}>{value.title}</h3>
```

Comment:
using styles.something is something we havent gone through in class yet, but i see you have figured it out. well done!